src/app/Containers/RecommendedMoviesContainer/index.unit.test.js

```diff
@@ -6,20 +6,29 @@ import { BrowserRouter as Router, Link } from 'react-router-dom';
 
 import MovieListContainer from '../MovieListContainer';
 import { MovieList } from '../../Components';
-import { getRecommended } from '../RecommendedMoviesContainer/duck/operations';
 import { SUCCESS } from '../../../common/constants';
 import Movie from '../../Components/Movie';
 
 const middlewares = [thunk];
 const mockStore = configureStore(middlewares);
 
+const operation = () => [
+  {
+    id: 1,
+    title: 'Black Mirror',
+
+    poster_url:
+      'https://m.media-amazon.com/images/M/MV5BYTM3YWVhMDMtNjczMy00NGEyLWJhZDctYjNhMTRkNDE0ZTI1XkEyXkFqcGdeQXVyMTkxNjUyNQ@@._V1_UX182_CR0,0,182,268_AL_.jpg',
+  },
+];
```

Comment:
This kind of mocking works, but it's not really necessary, since you're already mocking the store. We're not using this.

We should simply use `jest.fn()`, to make clear that it's a mock. (This applies to every operation mocked on this commit)